docs/source/api/v2/servers_hostname_update_status.rst

```diff
@@ -52,16 +52,26 @@ Response Structure
 ------------------
 Each object in the returned array\ [1]_ will contain the following fields:
 
+:configUpdateTime:     The last time an update was requested for this server. This field defaults to standard epoch
+:configApplyTime:      The last time an update was applied for this server. This field defaults to standard epoch
 :host_id:              The integral, unique identifier for the server for which the other fields in this object represent the pending updates and revalidation status
 :host_name:            The (short) hostname of the server for which the other fields in this object represent the pending updates and revalidation status
 :parent_pending:       A boolean telling whether or not the :term:`parents` of this server have pending updates
 :parent_reval_pending: A boolean telling whether or not the :term:`parents` of this server have pending :term:`Content Invalidation Jobs`
 :reval_pending:        ``true`` if the server has pending :term:`Content Invalidation Jobs`, ``false`` otherwise
-:status:               The name of the status of this server
+
+    .. note:: While not officially deprecated, this is based on the values corresponding to ``revalUpdateTime`` and ``revalApplyTime``. It is preferred to rely on the timestamp fields going forward as this will likely be deprecated in the future.
```

Comment:
Because we need it to work with potentially actively deployed T3C, we can't remove it outright in this PR. But I will deprecate it. For the time being we'll return both the old way (`updPending` and `revalPending`), but these values will be determined by the new timestamp values. The timestamp values will be returned in addition to, allowing for clients to transition between versions.

Reply:
`t3c` uses two different versions of our API client exactly so that we can do things like remove fields in new major API versions.